docs/intro.md

```diff
@@ -8,23 +8,26 @@ description: Intro to Rollkit, a modular framework for rollups.
 
 Welcome to Rollkit docs. We’re happy you made it here!
 
-Our mission is to empower developers to quickly innovate and create entire new classes of rollups with minimal tradeoffs.
-
-We’re setting the bar high for developers’ flexibility and ability to customize rollups however they see fit.
-
 :::tip Tip
 If you're familiar with Rollkit, you may want to skip to the [tutorials section](../category/tutorials)
 :::
 
 ## What is Rollkit?
 
-Rollkit is a rollup framework that gives developers the freedom to deploy rollups throughout the modular stack with minimal tradeoffs, opening new possibilities for rapid experimentation and innovation.
+Rollkit is a rollup framework that gives developers the freedom to deploy rollups throughout the modular stack, opening new possibilities for rapid experimentation and innovation.
+
+The Rollkit framework features a modular node that can run rollups and expose an **[ABCI](https://github.com/informalsystems/tendermint/tree/main/abci)**-compatible client interface, which can be used as a substitute for Tendermint in any ABCI-compatible blockchain application.\
+By default, the node utilizes Celestia as the DA layer. In the future, the node will be able to connect to any DA layer of choice.
 
-It was built as a drop-in replacement for Tendermint for any ABCI-compatible blockchain app.
+In addition to deploying sovereign app-rollups, Rollkit can also be used to deploy dapps on existing settlement and execution layers, or even to create a new settlement layer. 
+The framework's strength lies in its flexibility, allowing developers to customize their rollups as per their requirements.
 
-We’re building Rollkit as an open-source public good. While Celestia Labs will fund R&D, we see a future where the developer community contributes and owns Rollkit development.
+Rollkit is built as an open-source framework, so that developers could easily import existing modules into their applications. 
+We encourage developers to contribute to the development of Rollkit modules by adding new ones or improving existing ones. 
 
-The tutorials use Celestia as the default data availability layer, but developers will have the ability to swap Celestia with any data availability layer.
+Our goal is to empower developers to quickly innovate and create new classes of rollups with minimal trade-offs.
+
+We’re making deploying a new chain as easy as deploying a smart contract!
```

Comment:
isn't this what Celestia is for? should we change messaging on this maybe?

Reply:
Hmm, don't think so. Rollkit is the tool that will make that possible. You don't use Celestia to easily create rollups.